.gitignore

```diff
@@ -1,4 +1,5 @@
 .vscode/
+.DS_Store
```

Comment:
out of curiosity, why

Reply:
this kept showing up in my `git status` but i didn't dig very deep into why it happens. I've seen it come up when i run the python interpreter to verify my changes, but it could be something else related to macbooks

Reply:
hm interesting